### ISSUE TITLE
CSP reports

### ISSUE THREAD

**Issue by FredUK** [sir-dunxalot/ember-tooltips#37]
Hi there,

After installing this plugin I'm getting a lot of CSP reports on the console saying:
Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive. tooltip.js:210..

Is there any way to supress these messages without adding unsafe-eval to my CSP configuration?

![image](https://cloud.githubusercontent.com/assets/11888256/12482841/c6105628-c049-11e5-9e0b-055bff493a3d.png)

**Comment by sir-dunxalot** [comment]
I don't know off the top of my head but I will look into it.

**Comment by jamesarosen** [comment]
I just started noticing these as well. They seem to come from
```js
require.register("darsain-event/index.js", Function("exports, require, module",
"'use strict';\n\
\n\
...
```

**Comment by sir-dunxalot** [comment]
Good spot, James! Is that code in the NPM dependency for the Darsain tooltips library?

**Comment by jamesarosen** [comment]
It's [right here](https://github.com/sir-dunxalot/ember-tooltips/blob/43842d88435151fbdd68b5164076b76263e6bb29/vendor/tooltip/tooltip.js#L210), as well as in the [published source](http://darsa.in/tooltip/dist/tooltip.js).

**Comment by sir-dunxalot** [comment]
Got it. :+1: 

Thanks for opening the issue there. Let's keep track of that and I'll look into whether there is anything that can be done in this library.

**Comment by adam-knights** [comment]
Addon looks good but we can't take it live with this issue. I've pinged darsain on the issue opened by james - if we cant get a dependency fix on this what are the options?

**Comment by sir-dunxalot** [comment]
This has been fixed with release `1.0.0`: https://github.com/sir-dunxalot/ember-tooltips/releases/tag/1.0.0
